src/rule_definition_tools.jl

```diff
@@ -1,4 +1,13 @@
 # These are some macros (and supporting functions) to make it easier to define rules.
+using Base.Meta
+
+macro strip_linenos(expr)
+    strip_linenos(x::LineNumberNode) = nothing
+    strip_linenos(x::Expr) = Expr(x.head, map(strip_linenos, x.args)...)
+    strip_linenos(x) = x
+    esc(strip_linenos(expr))
```

Comment:
BlueStyle requires returns
```suggestion
    return esc(strip_linenos(expr))
```
though is the escaping required?
```suggestion
    return strip_linenos(expr)
```

Reply:
> though is the escaping required?

yes